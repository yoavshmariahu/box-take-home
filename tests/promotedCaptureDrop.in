k b2
K d4
+p e5
+P a1

[G S B R]
[r b s g]

move b2 a1
move d4 e5
drop p b1
drop p b5
