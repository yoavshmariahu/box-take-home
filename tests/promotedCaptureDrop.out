UPPER player action: drop p b5
5 |__| P|__|__| K|
4 |__|__|__|__|__|
3 |__|__|__|__|__|
2 |__|__|__|__|__|
1 | k| p|__|__|__|
    a  b  c  d  e

Captures UPPER: G S B R
Captures lower: r b s g

lower> 
